### ISSUE TITLE
VLCTube seems to mostly work in all versions of firefox up to nightly 40.0a1 with some issue

### ISSUE THREAD

**Issue by Sonic0170** [jackun/VLCTube#38]
so i decided to do a test in firefox nightly for the hell of it and VLCTube won't load at all, im guessing more security has been implemented or changed.
I tried looking into the console and through the page source but i couldn't find any instances of the script loading or erroring out.
it works in stable firefox build v34 100% which is my default browser im using at this time.

**Comment by Sonic0170** [comment]
i did another test and same thing but its 38.0a1 now, any idea whats happening?

**Comment by jackun** [comment]
Seems that I'd have to hardcode settings variable names, for some time to come atleast.

**Comment by Sonic0170** [comment]
ah i see, well no hurry since i stopped using nightly, at least you know what may be happening ^.^

**Comment by asampal** [comment]
@Sonic0170 just wanted to let you know that up to v36 beta 10 it's still working for me.

**Comment by Sonic0170** [comment]
yes im aware @asampal , its v37 and higher that has changed some security that forces jackun to rewrite a bunch of the script to make it work again.
since the stable build is v35 right now, its gonna be a good while before his hand is forced.

**Comment by Sonic0170** [comment]
so i did a test and had v38.0a1 and it works fine now, but i then updated to v39.0a1 and it doesn't load at all.

this is just informational and is not a priority, it will be a while before we get to v39 firefox

**Comment by Sonic0170** [comment]
after recent tests, VLCTube is confirmed to be working on all versions of Firefox up to nightly version v40.0a1 but experiences occasional lockups that require Firefox to be force crashed to resolve, it is currently unknown why this happens.
